main/tests/test_gdscript.cpp

```diff
@@ -318,6 +318,7 @@ static void _parser_show_block(const GDParser::BlockNode *p_block,int p_indent)
 						else
 							_print_indent(p_indent,"return ");
 					} break;
+					case GDParser::ControlFlowNode::CF_MATCH: break;
```

Comment:
That one is wrong, it should "unparse" the `match` statement, instead of just covering the issue in a break. If you think it would be too complex for the scope of this PR, you should either leave a `TODO` comment, or not change it at all.